docs/organizations/segments.md

```diff
@@ -0,0 +1,36 @@
+# Segments
+Segments are dimensions that Codacy reads from your provider that organizes repositories into relevant groups for you. Today, Segments is available for: [GitHub Custom Properties](#github-custom-properties)
+  
+
+### Where can Segments be utilised?
+- [Repository list](../managing-repositories/#provider-segments) 
+- [Security & Management Risk](../managing-security-and-risk/)
+
+### Enabling Segments {: id="enabling-segments"}
+To enable Segments, an initial sync between your provider and Codacy needs to happen. Once completed, you can use Segments to better locate and organize repositories within Codacy.
+![Repositories list segments sync](../organizations/images/security-risk-management-segments-overview.png)
```

Comment:
I wonder if it would make sense in this "Enabling Segments" section to have a first print screen with the feature not yet enabled giving emphasis to the "Github Segments" button. 
![Screenshot 2024-10-10 at 11 28 33](https://github.com/user-attachments/assets/6cb8151e-195f-4b78-9e43-ebd3ea52da0e)

Then the images would reflect a *before* and *after* state.